include/avif/avif.h

```diff
@@ -537,8 +539,10 @@ typedef struct avifReformatState
     float rgbMaxChannelF;
     float biasY;   // minimum Y value
     float biasUV;  // the value of 0.5 for the appropriate bit depth [128, 512, 2048]
+    float biasA;   // minimum A value
```

Comment:
Hi Yuan Tong: In https://github.com/AOMediaCodec/libavif/pull/515 I propose to remove avifReformatMode, avifReformatState, and avifPrepareReformatState() from the public API. If that pull request is accepted, the definition of struct avifReformatState will be moved to src/reformat.c or include/avif/internal.h. Please follow that pull request. Thanks!

Note: Based on your changes to reformat_libyuv.c in this pull request, I think I should move the definition of struct avifReformatState to include/avif/internal.h.